src/walk-manifest.js

```diff
@@ -202,9 +200,9 @@ var walkPlaylist = function(options, callback) {
 
       // SUB Playlists
       // The reflect is used so we can still continue running even if one of the playlists is broken.
-      async.map(playlists, async.reflect(function(p, cb) {
+      async.map(playlists, function(p, cb) {
         if (!p.uri) {
-          return cb(null);
+          return cb(null, null);
```

Comment:
might be nice to return `[]` instead of null (not sure if it matters with the code involved).